libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/internal/route/line/CacheResultUtils.kt

```diff
@@ -0,0 +1,43 @@
+package com.mapbox.navigation.ui.maps.internal.route.line
+
+import android.util.LruCache
+
+internal object CacheResultUtils {
```

Comment:
Could you add unit tests?

Reply:
I understand some of it is covered by the route line tests, but it doesn't covers the specific of the cache itself like overflow handling etc.

Reply:
Yep, I wanted to make sure there were no objections to this approach before doing the work of adding unit tests for these methods.  I will add them.

Reply:
testing added